### DISCUSSION TITLE
How can I save the best model with a minimun epoch?

### DISCUSSION THREAD

**Discussion by xianyuanliu** [Lightning-AI/pytorch-lightning#6499] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi all,

In my case, I hope model training for 10 epochs before adding an extra loss, and I need to get the best model for the rest epochs. Can I use `ModelCheckpoint` to do it? Many thanks!
